dataeng/resources/exporter-worker.sh

```diff
@@ -0,0 +1,14 @@
+TODAY=$(date +%d)
+
+env | sort
+
+${EXPORTER_VENV}/bin/exporter \
+   ${EXTRA_OPTIONS} ${CONFIG_PATH} ${ORG_CONFIG_PATH} \
+   --org=${ORG} \
+   --task=OrgEmailOptInTask \
```

Comment:
Just wondering why this particular task is in a file named "exporter-worker.sh"?    Is there a more specific filename that should be used?

Reply:
Would 'email_optin_worker' be more appropriate ?

Reply:
Yeah we probably want to make this named email opt-in specific.

Reply:
I was assuming we would have the regular export moving to this box as well.  This name would have been misleading (to me).